batch/batch_allen.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
```

Comment:
These files for launching slurm jobs should probably be hidden/removed.

Reply:
I agree! Just removed them.